src/config.js

```diff
@@ -17,6 +18,14 @@ const Config = CoreObject.extend({
    * @param {String} attrs.myShopifyDomain You shop's `myshopify.com` domain.
    */
   constructor(attrs) {
+    Object.keys(this.deprecatedProperties).forEach(key => {
+      const transformName = this.deprecatedProperties[key];
```

Comment:
could move this stuff within the conditional so you're only doing it if there is a deprecated property